react/src/components/ImageInstallModal.tsx

```diff
@@ -0,0 +1,197 @@
+import { useSuspendedBackendaiClient } from '../hooks';
+import { useSetBAINotification } from '../hooks/useBAINotification';
+import { usePainKiller } from '../hooks/usePainKiller';
+import BAIModal, { BAIModalProps } from './BAIModal';
+import Flex from './Flex';
+import { EnvironmentImage } from './ImageList';
+import { App, List } from 'antd';
+import _ from 'lodash';
+import { Dispatch, Key, SetStateAction } from 'react';
+import { useTranslation } from 'react-i18next';
+
+interface ImageInstallModalInterface extends BAIModalProps {
+  open: boolean;
+  onRequestClose: () => void;
+  selectedRowKeys: Key[];
+  images: EnvironmentImage[];
+  setInstallingImages: Dispatch<SetStateAction<string[]>>;
+  setSelectedRowKeys: Dispatch<SetStateAction<Key[]>>;
+}
```

Comment:
If you declare a new interface by extending antd's default modal, the fields included in the existing modal interface can be removed from the declaration and also other fields in the main modal interface are also available.[[related comment]](https://app.graphite.dev/github/pr/lablup/backend.ai-webui/2615/feat-migrate-image-list#comment-PRRC_kwDOCRTcws5mHAhD) [[related comment]](https://app.graphite.dev/github/pr/lablup/backend.ai-webui/2615/feat-migrate-image-list#comment-PRRC_kwDOCRTcws5mHAoW) [[related comment]](https://app.graphite.dev/github/pr/lablup/backend.ai-webui/2615/feat-migrate-image-list#comment-PRRC_kwDOCRTcws5mHAtJ)